internal/quickstart/messages.go

```diff
@@ -117,6 +136,7 @@ func createFlag(client flags.Client, accessToken, baseUri, flagName, flagKey, pr
 					),
 				}
 			}
+			existingFlag = true
```

Comment:
I realized that with the recent changes #179 we were now forgetting to set `existingFlag` when there is a createFlag conflict